docs/processes/documentation/documentation.md

```diff
@@ -0,0 +1,37 @@
+## Introduction
+
+Documentation is one of the most important parts of any software epic. It is imperative to remember, you will not be working on the capstone projects forever and that there will be people taking your place within Thoth Tech.
```

Comment:
I would prefer if our handbook was less capstone related and more company focused.
Although, Thith Tech is for capstone, it is still a viable open source company and should be spoken about as such.

Consider rephrasing: 

```suggestion
Thoth Tech prides itself on our ability to produce sustainable solutions, this can be achieved through documentation.
Documentation is an important element in the software development life cycle. It continues throughout the entire project.
It is imperative to remember, others will take over your projects or want to learn more about how it works and why. 
```